index.html

```diff
@@ -5,30 +5,67 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
     <title>Document</title>
 </head>
 
+
 <body>
-    <h1>Roberto Gavino Flores</h1>
+    <header>
```

Comment:
Nav header should include all sections (message, connect).  This is for 4.5 though.